assets/stablelm.yaml

```diff
@@ -0,0 +1,23 @@
+---
+
+- type: model
+  name: StableLM
+  organization: Stability AI
+  description: Large language models trained on up to 1.5 trillion tokens.
+  created_date: 2023-04-20
+  url: https://github.com/Stability-AI/StableLM
+  model_card: ''
+  modality: natural language text
+  analysis: ''
+  size: 7B parameters
+  dependencies: [The Pile, Stanford Alpaca]
```

Comment:
I don't think this is right. I think the dependencies here are:

1. StableLM-Alpha dataset, which itself should have a dependency on The Pile
2. Alpaca dataset
3. gpt4all dataset
4. ShareGPT52K dataset
5. Dolly dataset
6. HH dataset

In particular, go to those other datasets and make sure you refer to them in the same way as they are annotated, so the graph looks correct/indicates what is shared.